apps/prairielearn/src/components/ProgressCircle.html.ts

```diff
@@ -48,7 +43,7 @@ export function ProgressCircle({
     width="20px"
     height="20px"
     viewBox="0 0 20 20"
-    class="mx-1 ${mlAuto ? 'ml-auto' : ''}"
+    class="${className ? className : ''}"
```

Comment:
Nit, this is slightly terser:

```suggestion
    class="${className ?? ''}"
```